### ISSUE TITLE
Floating instance for behaviors?

### ISSUE THREAD

**Issue by noughtmare** [reflex-frp/reflex#465]
I think this instance should work:

```haskell
instance (Reflex t, Floating a) => Floating (Behavior t a) where
  pi = pure pi
  exp = fmap exp
  log = fmap log
  sin = fmap sin
  cos = fmap cos
  asin = fmap asin
  acos = fmap acos
  atan = fmap atan
  sinh = fmap sinh
  cosh = fmap cosh
  asinh = fmap asinh
  acosh = fmap acosh
  atanh = fmap atanh
```

**Comment by Ericson2314** [comment]
I am not so sure this is a good idea? This is true for any Applicative, and I prefer to be explicit about operation on the side vs on the functor "container".

**Comment by noughtmare** [comment]
There are already similar instances for `Semigroup`, `Monoid`, `Num`, and `Fractional`, so to me it's strange to draw the line here.

**Comment by noughtmare** [comment]
I also noticed that there are `Semigroup`, `Monoid` and `Num` instances for `Dynamic` but no `Fractional` or `Floating` instances.

**Comment by mrehayden1** [comment]
Yes, `Fractional a => Fractional (Dynamic t a)` would be nice to be able to shorthand `pure 0.1` or `constDyn 0.1` with just `0.1`.
